app/services/eligibility_service.rb

```diff
@@ -101,20 +110,18 @@ def presence_values
     presence
   end
 
-  def years_before(num_years)
-    (@day.to_date - num_years.year).strftime(date_format)
+  def years_before(day, num_years)
+    (day.to_date - num_years.year).strftime(EligibilityService.date_format)
   end
 
-  def date_format
+  def self.date_format
     '%Y-%m-%d'
   end
 
-  def person_name
-    'Ruby'
-  end
-
   def calculate(query)
+    Rails.logger.info 'Calculating'
     Rails.cache.fetch(query) do
+      Rails.logger.info 'Requesting from OpenFisca'
```

Comment:
Are you meaning to keep all of these logs in the server? There are quite a few throughout this PR.